cmd/serve.go

```diff
@@ -72,6 +73,14 @@ func serve(ctx context.Context) {
 		defer auditCloseFn() //nolint:errcheck // Not needed to check returned error.
 	}
 
+	perms, err := permissions.New(config.Config.Permissions,
+		permissions.WithLogger(logger),
+		permissions.WithDefaultChecker(permissions.DefaultAllowChecker),
```

Comment:
We should probably make this configurable (with the default deny checker as the default).

Reply:
Indeed I see what you're saying. Let me see about adding this configuration into the permissions library as I think that would be a much needed improvement to that and simplify these downstream app integrations.

Reply:
I've created this ability in this PR for permissions-api: https://github.com/infratographer/permissions-api/pull/217

Reply:
Sounds good! We can probably just remove this line then.